Sources/MapboxNavigation/NavigationMapView+BuildingHighlighting.swift

```diff
@@ -59,57 +59,53 @@ extension NavigationMapView {
     private func addBuildingsLayer(with identifiers: Set<Int64>, in3D: Bool = false, extrudeAll: Bool = false) {
         let identifier = NavigationMapView.LayerIdentifier.buildingExtrusionLayer
         
-        do {
```

Comment:
I think building highlighting layer addition was resolved [here](https://github.com/mapbox/mapbox-navigation-ios/pull/3010/files#diff-e6fd431d44128353bd17514840125bf6079a540908893a32297f42a9bde222faR68-R70). So probably rebasing onto that PR would be better, scope of this PR is a bit different anyway. It'll prevent merge conflicts as well.

Reply:
Thanks for the suggestion, I'll remove this part.